jobs/build/ocp/Jenkinsfile

```diff
@@ -144,7 +144,11 @@ node(TARGET_NODE) {
         }
 
         stage( "openshift-jenkins repo ") {
-          buildlib.initialize_openshift_jenkins()
+          JENKINS_SOURCE_BRANCH = "master"
+          if(RELEASE_MINOR > 6 && MASTER_RELEASE != MAJOR_RELEASE){
```

Comment:
None of these are defined. You''ll need to implement a pattern similar to openshift-ansible prep.